extensions/markdown-math/syntaxes/md-math-block.tmLanguage.json

```diff
@@ -79,6 +82,42 @@
 					]
 				}
 			]
+		},
```

Comment:
For the grammar changes, I think we should create a separate injection grammar instead that uses this approach: https://github.com/steebn/vscode-fenced-code-block-grammar-injection-example/tree/master

It should reduce some of the grammar duplication and make sure we only render fenced code blocks in expected places